scripts/js/commands/checkImages.ts

```diff
@@ -24,19 +56,38 @@ async function main() {
 
     if (imageErrors.size) {
       fileErrors.push(
-        `Error in file '${file}':\n\t- ${[...imageErrors].join("\n\t- ")}`,
+        `Error in file '${file}':\n\t- ${[...imageErrors].join("\n\t- ")}\n`,
       );
     }
   }
 
   if (fileErrors.length) {
     fileErrors.forEach((error) => console.log(error));
     console.error(
-      "\n💔 Some images have problems. See https://github.com/Qiskit/documentation#images for instructions.\n",
+      "💔 Some images have problems. See https://github.com/Qiskit/documentation#images for instructions.\n",
     );
     process.exit(1);
   }
   console.log("✅ All images are valid.\n");
 }
 
+async function determineTocFiles(args: Arguments): Promise<string[]> {
+  // We always skip historical versions to simplify the code and to have a faster script.
+  // Even though it is possible for someone to add a new image without alt text to a
+  // historical version that wasn't in the original release, that's very unlikely.
+  // If it happens, it would probably be a backport from latest or dev, and the linter in
+  // the API repo should catch it.
+  //
+  // If an image is missed by the API repo's linter, it will still have an alt text defined,
+  // although it won't be very useful.
+  const globs = [
+    "docs/**/*.{ipynb,mdx}",
+    "!docs/api/qiskit/release-notes/*.mdx",
+    args.apis ? "!docs/api/*/([0-9]*)/*.mdx" : "!docs/api/**/*.mdx",
```

Comment:
Should we check release notes with `--apis`? If it's slow then we could ignore anything < 1.0

```
!docs/api/qiskit/release-notes/0.*.mdx
```

Reply:
Yeah, I think it's a good idea to check the release notes. I've checked and it's not adding a lot of extra time so it's probably okay to check all of them. Thanks!